deploy/gk-deploy

```diff
@@ -75,6 +77,17 @@ Options:
               device or node, then restart with --load to skip gluster
               deployment and re-run topology load command.
 
+  --admin-key ADMIN_KEY
+              Secret for heketi administrator user. Heketi admin has access to all 
+	      the APIs. admin key: string, Shared secret. 
+	      Eg: MySecret
+
+
```

Comment:
Make sure all lines have the same indentation. Also, have only one newline (instead of two) between blocks of text.